### DISCUSSION TITLE
Can I stick/fix a cell on the screen as a preview

### DISCUSSION THREAD

**Discussion by Roy-Kid** [marimo-team/marimo#1616] [Ideas]
Hi, I am developing an molecular visualization widget with python REPL control. I want a cell to be fixed so I can continue executing commands in other cells to control the widget. Here is a demo screenshot implemented by using stickyland(https://github.com/xiaohk/stickyland): 

<img width="1917" alt="695faa65b35bbec689ef095940e1eac" src="https://github.com/marimo-team/marimo/assets/42854324/7e3068a3-7871-4de5-8efd-b86a2dffa7f1">

**Comment by dmadisetti** [comment]
Similar to the "scratch pad" proposal: https://github.com/marimo-team/marimo/pull/1475

But yeah, I think this would be nice too

**Reply by Roy-Kid** [reply]
Thanks for your info! It's quite useful!!!

**Comment by akshayka** [comment]
@Roy-Kid -- sorry, I missed this. I also think it would be nice! 

Alternatively, once we support multiple browser tabs connected to the same session ("multiplayer"), then you could open two browser windows side-by-side, showing the same notebook but at different scrolls. I often do this with other tools.

**Reply by Roy-Kid** [reply]
That sounds good! I also like multi-windows work style.

But it would be better(or easy to implement) a sticky cell for small note / result? Just an advice : )
